### PR TITLE
[WIP] Add blog article for In-place Pod Resize feature (v1.26 alpha)

### PR DESCRIPTION
**Pull request by vinaykul**
This is a place holder blog PR for In-place pod resize feature that is targeted for v1.26

### CODE REVIEW

**Comment by sftim** [comment]
Actually
/retitle Add blog article for In-place Pod Resize feature (v1.26 alpha)

**Comment by sftim** [comment]
:facepalm:
 /retitle [WIP] Add blog article for In-place Pod Resize feature (v1.26 alpha)

**Comment by jihoon-seo** [comment]
/retitle [WIP] Add blog article for In-place Pod Resize feature (v1.26 alpha)

File: content/en/blog/_posts/2022-10-21-in-place-pod-resize-alpha.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: "Kubernetes 1.26: In-place Resource Resize for Kubernetes Pods lands as alpha"
+date: 2022-10-21
+slug: in-place-pod-resize-alpha
+---
+
+**Author:** Vinay Kulkarni (Futurewei Technologies)
+
+TBD-placeholder
+
+## What's new in 1.25?
```
Comment (sftim) [inline content/en/blog/_posts/2022-10-21-in-place-pod-resize-alpha.md:12]: nit: this'll need to change (v1.26)

**Comment by fsmunoz** [comment]
Hello, Comms Lead for v.126 here. This enhancement was opted-in for a Feature Blog, but the enhancement was removed from milestone (https://github.com/kubernetes/enhancements/issues/1287), so we aren't tracking it for publication. If it is included in a future release then a new PR can be created during the feature opt-in tracking process of that release. Thanks!

**Comment by fsmunoz** [comment]
Hello @vinaykul  , we're doing a global reminder about submitting a draft for review for all opted-in feature blogs. If it's at all possible, it is very helpful for the release team to have drafts submitted for review before the hard deadline date, to better plan the release dates and avoid missing out. Thank you!

**Comment by vinaykul** [comment]
> Hello @vinaykul , we're doing a global reminder about submitting a draft for review for all opted-in feature blogs. If it's at all possible, it is very helpful for the release team to have drafts submitted for review before the hard deadline date, to better plan the release dates and avoid missing out. Thank you!

@fsmunoz This feature didn't make it into 1.26 release. I'm hoping to bring the code early into 1.27. Please stop tracking this towards 1.26 release blog and retarget to 1.27. Thanks,

**Comment by tengqm** [comment]
/close
Let's target 1.27 then.

**Pull request closed without merging**